extensions/amp-story/1.0/amp-story.js

```diff
@@ -989,13 +989,27 @@ export class AmpStory extends AMP.BaseElement {
             'amp-story-page-attachment, amp-story-page-outlink'
           );
 
-          if (attachmentEl) {
+          if (
+            attachmentEl &&
+            attachmentEl.tagName !== 'AMP-STORY-SHOPPING-ATTACHMENT'
```

Comment:
Do you still need this line, given that the selector can't query this specific tag?

Reply:
nope, removed